maestro/demos/workflows/cbom.ai/remediation/README.md

```diff
@@ -0,0 +1,156 @@
+# QSC Remediation
+
+The demo consists of three agents:
+
+| Agent | Purpose | In | Out | Tools | Notes |
+| -- | -- | -- | -- | -- | -- |
+|Finder|Parse CBOM to look for specific QSC issues to fix|CBOM|JSON findings|-|The mastercard CBOM from sonarqube scan is used|
+|Fixer|Fix code to make it safer|JSON findings|git email patch|fixer||
+|Patcher|Fix code to make it safer|git email patch|URL to PR|patcher||
+
+The Agents will together take a CBOM scan of a github repository, and by the end will have raised a pull request against
+the repo with a suggested fix.
+
+## Prereqs
+
+* Python 3.13
+* `pip install -r requirements.txt -U`
+* Agents are written to use the *BAM* runtime with *meta-llama/llama-3-1-70b-instruct* (see example config below)
+
+## Executor image
+
+This demo requires the following tools to be available within the executor environment:
+
+* git
+* gh (GitHub cli)
+
+### Building the image (optional)
+
```

Comment:
Kind of hate this. We really need to move to using GH tools MCP. I know in future version...

Reply:
Exactly. Couldn't agree more. At the time the original demo was made we didn't have MCP.
I ported the existing agents, but had troubles with the tools (dependency changes). Using a better tool setup (ie MCP) is essential - that + defining a new workflow is next step.